cli/command/service/logs.go

```diff
@@ -49,13 +49,14 @@ func newLogsCommand(dockerCli *command.DockerCli) *cobra.Command {
 			opts.target = args[0]
 			return runLogs(dockerCli, &opts)
 		},
-		Tags: map[string]string{"experimental": ""},
```

Comment:
Right now, if a current client talks to an older daemon with experimental enabled, the API is relatively unchanged, so the behavior might not be what you're expecting, but it will probably work.